src/main/java/pageobject/LoginPage.java

```diff
@@ -1,79 +1,93 @@
 package pageobject;
 
 
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
+//import org.openqa.selenium.WebElement;
+//import org.openqa.selenium.remote.server.handler.ClickElement;
+//import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
+import Base.BaseActions;
 
-public class LoginPage {
 
-	
-	@FindBy(id="user_email")
-	WebElement email;
-	
-	@FindBy(id="user_password")
-	WebElement password;
+public class LoginPage extends BaseActions
+{
 
-	@FindBy(name="commit")
-	WebElement submit;
+	WebDriver  driver;
 	
-	@FindBy(xpath="//div[contains(@class,'alert-danger')]")
-	WebElement errorMsg;
+	By home_login = By.xpath("//button[@class='a-button f-b pure-input-1 f-b-4A90E2-a f-b-4A90E2-h f-border-4A90E2 f-c-FFFFFF-a f-c-FFFFFF-h f-c-4A90E2 f-l-38 f-m-a f-r-20 f-s-16 f-w-90 f-600']");
+	By email = By.id("tooltip_auth_login_email");
+	By password = By.id("tooltip_auth_login_password");
+	By submit = By.xpath("//button[@class='a-button f-b f-b-293B50-a f-b-293B50-h f-b-4A90E2 f-c-FFFFFF f-h-40 f-l-40 f-m-a f-p-10-h f-r-20 f-s-14 f-w-175 f-600 pure-input-1'][text()='Log In']");
```

Comment:
//button[@class='a-button f-b f-b-293B50-a f-b-293B50-h f-b-4A90E2 f-c-FFFFFF f-h-40 f-l-40 f-m-a f-p-10-h f-r-20 f-s-14 f-w-175 f-600 pure-input-1'][text()='Log In']"
this can written as
//button[text()='Log In'] no need to add class names.

Reply:
Changed Xpath